postgres_ffi/build.rs

```diff
@@ -19,20 +57,23 @@ fn main() {
         // Tell cargo to invalidate the built crate whenever any of the
         // included header files changed.
         //
-        .parse_callbacks(Box::new(bindgen::CargoCallbacks))
+        .parse_callbacks(Box::new(PostgresFfiCallbacks))
```

Comment:
I'm a bit surprised we need to use the callback for this. There are `derive_copy`, `derive_debug` etc. functions in `Builder`, but no `derive_custom` to add arbitrary derives. Ah well.